graylog2-web-interface/src/components/welcome/RecentActivityList.tsx

```diff
@@ -0,0 +1,107 @@
+ */
+
+import React, { useCallback, useMemo, useState } from 'react';
+import styled, { css } from 'styled-components';
+
+import { Table } from 'components/bootstrap';
+import { DEFAULT_PAGINATION, entityTypeMap } from 'components/welcome/Constants';
+import { NoSearchResult, PaginatedList, Spinner } from 'components/common';
+import { relativeDifference } from 'util/DateTime';
+import Routes from 'routing/Routes';
+import { Link } from 'components/common/router';
+import { StyledLabel } from 'components/welcome/EntityListItem';
+import { useRecentActivity } from 'components/welcome/hooks';
+import type { EntityItemType, RecentActivityType } from 'components/welcome/types';
+
+const ActionItemLink = styled(Link)(({ theme }) => css`
+  color: ${theme.colors.variant.primary};
+  &:hover {
+    color: ${theme.colors.variant.darker.primary};
+  }
+`);
+
+type Props = { itemType: EntityItemType, itemId: string, activityType: RecentActivityType, itemTitle: string, userName?: string };
+
+const ActionItem = ({ itemType, itemId, activityType, itemTitle, userName }: Props) => {
+  const { typeTitle } = entityTypeMap[itemType];
+  const entityLink = useMemo(() => {
+    return entityTypeMap[itemType] ? Routes.pluginRoute(entityTypeMap[itemType].link)(itemId) : undefined;
+  }, [itemId, itemType]);
+
+  return (
+    <div>
+      {`The ${typeTitle} `}
+      {activityType === 'delete'
+        ? <i>{itemTitle || itemId}</i>
+        : <ActionItemLink target="_blank" to={entityLink}>{itemTitle || itemId}</ActionItemLink>}
+      {' was '}
+      {`${activityType}d`}
+      {userName ? ` by ${userName}` : ''}
+    </div>
+  );
+};
+
+ActionItem.defaultProps = {
+  userName: null,
+};
+
+const RecentActivityList = () => {
+  const [pagination, setPagination] = useState(DEFAULT_PAGINATION);
+  const { data: { recentActivity, total }, isFetching } = useRecentActivity(pagination);
+  const onPageChange = useCallback((newPage) => {
+    setPagination((cur) => ({ ...cur, page: newPage }));
+  }, [setPagination]);
+
+  if (isFetching) return <Spinner />;
+
+  if (recentActivity.length === 0) {
+    return (
+      <NoSearchResult>
+        There is no recent activity yet.
+        <p />
+        Whenever any other user will update content you have access to, or share new content with you, it will show up here.
+      </NoSearchResult>
+    );
+  }
+
+  return (
+    <PaginatedList onChange={onPageChange} useQueryParameter={false} activePage={pagination.page} totalItems={total} pageSize={pagination.per_page} showPageSizeSelect={false} hideFirstAndLastPageLinks>
+      <Table striped>
+        <tbody>
+          {
+              recentActivity.map(({ id, timestamp, activityType, itemType, itemId, itemTitle, userName }) => {
+                return (
+                  <tr key={id}>
+                    <td style={{ width: 110 }}>
+                      <StyledLabel title={timestamp} bsStyle="primary">{relativeDifference(timestamp)}
```

Comment:
We have component `RelativeTime` which can be used in this case. Have you considered using the component?  I saw that you added the timestamp as a `title`, which is great. But with the react component we are able to adjust all the format of all titles for relative times in one place.